Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_External_-_Generic_v2.yml

```diff
@@ -252,6 +252,24 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+    fieldMapping:
+    - incidentfield: Detected External IPs
```

Comment:
This field is associated only with the PANW Endpoint Malware type. 
Maybe we should change that accordingly? 
or use extended context?

![image](https://github.com/demisto/content/assets/88268646/25c6ace5-4485-4355-876b-0967dab70ab6)

Reply:
I'll remove it, just thought it'd be nice